tikv/server.go

```diff
@@ -591,18 +592,7 @@ func (svr *Server) BatchCoprocessor(req *coprocessor.BatchRequest, batchCopServe
 		if reqCtx.regErr != nil {
 			return &RegionError{err: reqCtx.regErr}
 		}
-		var mppTaskHandler *cophandler.MPPTaskHandler
-		if mockRegionRM, ok := svr.regionManager.(*MockRegionManager); ok {
-			mppTaskHandlerMap := mockRegionRM.getMPPTaskSet(reqCtx.storeId)
-			if mppTaskHandlerMap != nil {
-				if th, ok := mppTaskHandlerMap[int64(req.Context.TaskId)]; ok {
-					mppTaskHandler = th
-				}
-			}
-		}
-		copResponse := cophandler.HandleCopRequestWithMPPCtx(reqCtx.getDBReader(), svr.mvccStore.lockStore, &cop, &cophandler.MPPCtx{
```

Comment:
So the `MPPCtx` is not needed?

Reply:
Yeah, "Batch Coprocessor" has no bessiness of MPP, I think that's a mistake of previous design.